app/src/main/java/com/github/se/cyrcle/ui/report/ViewReportsScreen.kt

```diff
@@ -50,22 +56,27 @@ fun ViewReportsScreen(
         TopAppBar(
             navigationActions,
             title =
-                if (selType == ReportedObjectType.PARKING) {
-                  stringResource(R.string.parking_reports)
-                } else {
-                  stringResource(R.string.review_reports)
+                when (selType) {
+                  ReportedObjectType.PARKING -> stringResource(R.string.parking_reports)
+                  ReportedObjectType.REVIEW -> stringResource(R.string.review_reports)
+                  else -> stringResource(R.string.image_reports)
                 })
       },
       floatingActionButton = {
         FloatingActionButton(
             onClick = {
               val uidOfObject = currentObject?.objectUID
               if (uidOfObject != null) {
+                // Delete the reported object and navigate back to the admin screen
                 reportedObjectViewModel.deleteReportedObject(uidOfObject)
-                if (selType == ReportedObjectType.PARKING) {
-                  parkingViewModel.deleteParkingByUid(uidOfObject)
-                } else {
-                  reviewViewModel.deleteReviewById(uidOfObject)
+                when (selType) {
+                  ReportedObjectType.PARKING -> parkingViewModel.deleteParkingByUid(uidOfObject)
+                  ReportedObjectType.REVIEW -> reviewViewModel.deleteReviewById(uidOfObject)
+                  ReportedObjectType.IMAGE -> {
+                    val uidOfParking = parkingViewModel.getParkingFromImagePath(uidOfObject)
+                    Log.d("$uidOfParking", "$uidOfObject")
```

Comment:
Could we not use a constant log tag? It would be easier to identify logs

Reply:
I don't think it's even necessary in retrospect